README.md

```diff
@@ -1,19 +1,19 @@
-# Victoria Secrets
+# Victoria Treasure
 
 ![Victoria](imgs/victoria.png)
-![Secrets](imgs/secrets.gif)
+![Treasure](imgs/Treasure.png)
 
 [![CircleCI](https://circleci.com/gh/peertransfer/VictoriaSecrets.svg?style=svg)](https://circleci.com/gh/peertransfer/VictoriaSecrets)
```

Comment:
What about here?

Reply:
what do you mean?

Reply:
That is retro compatible , but I have to change it you are right